misk/src/main/kotlin/misk/hibernate/RealTransacter.kt

```diff
@@ -39,6 +59,16 @@ internal class RealTransacter(
     }
   }
 
+  override fun retries(numRetries: Int): Transacter {
```

Comment:
can we always specify things in number of attempts rather than in number of retries? If the code says retries(3) I’m going to implicitly think 3 attempts, not 4 attempts.

Reply:
also – should we impose some simplicity on the caller here? Perhaps “retries” and “no retries” should be our only options. I don’t like the idea of somebody specifying 2 attempts.